src/Umbraco.Cms.ManagementApi/ManagementApiComposer.cs

```diff
@@ -0,0 +1,131 @@
+﻿using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Diagnostics;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Versioning;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+using NSwag.AspNetCore;
+using Umbraco.Cms.Core;
+using Umbraco.Cms.Core.Composing;
+using Umbraco.Cms.Core.Configuration.Models;
+using Umbraco.Cms.Core.DependencyInjection;
+using Umbraco.Cms.Core.Hosting;
+using Umbraco.Cms.ManagementApi.DependencyInjection;
+using Umbraco.Cms.Web.Common.ApplicationBuilder;
+using Umbraco.Extensions;
+using Umbraco.New.Cms.Web.Common.Routing;
+
+namespace Umbraco.Cms.ManagementApi;
+
+public class ManagementApiComposer : IComposer
+{
+    private const string ApiTitle = "Umbraco Backoffice API";
+    private const string ApiAllName = "All";
+
+
+    public void Compose(IUmbracoBuilder builder)
+    {
+        IServiceCollection services = builder.Services;
+
+        builder
+            .AddNewInstaller()
+            .AddUpgrader();
+
+        services.AddApiVersioning(options =>
+        {
+            options.DefaultApiVersion = new ApiVersion(1, 0);
```

Comment:
Move these version numbers to consts, or create a private method to return the default `ApiVersion` --- and reuse it below in `AddVersionedApiExplorer`